src/java.rmi/share/classes/sun/rmi/server/LoaderHandler.java

```diff
@@ -536,96 +338,27 @@ public static Class<?> loadProxyClass(String codebase, String[] interfaces,
         }
 
         /*
-         * If no security manager is set, disable access to RMI class
+         * There is no security manager, so disable access to RMI class
          * loaders and use the would-de parent instead.
```

Comment:
Fix typo "would-de" -> "would-be".

Reply:
Fixed in https://github.com/openjdk/jdk/pull/21498/commits/0f448e5fabac7941cd1b551aa1cc9ade773814ff